src/schemas/execute.yaml

```diff
@@ -0,0 +1,347 @@
+    - gasUsed
+    - logs
+  properties:
+    status:
+      title: Call Status Success
+      type: string
+      pattern: ^0x1$
+    returnData:
+      title: Return data
+      $ref: '#/components/schemas/bytes'
+    gasUsed:
+      title: Return gasUsed
+      $ref: '#/components/schemas/uint64'
+    logs:
+      title: Return logs
+      type: array
+      items:
+        $ref: '#/components/schemas/CallResultLog'
+CallResultInvalid:
+  title: Result of call not being valid (nonce, baseFeePerGas, etc)
+  description: The error messages are suggestions, and clients might implement different error messages. However, the error codes are enforced by the spec.
+  type: object
+  required:
+    - status
+    - error
+  properties:
+    status:
+      title: Call Status Invalid
+      type: string
+      pattern: ^0x2$
+    error:
+      oneOf:
+        - code: -32000
+          message: Missing or invalid parameters
+        - code: -32005
+          message: Limit exceeded
+        - code: -32015
+          messagE: Execution error
+        - code: -32016
+          message: Timeout
+        - code: -32602
+          message: Invalid params
+        - code: -32603
+          message: The Ethereum node encountered an internal error
+        - code: -38010
+          message: Transactions nonce is too low
+        - code: -38011
+          message: Transactions nonce is too high
+        - code: -38012
+          message: Transactions baseFeePerGas is too low
+        - code: -38013
+          message: Not enough gas provided to pay for intrinsic gas for a transaction
+        - code: -38014
+          message: Insufficient funds to pay for gas fees and value fo a transaction
+        - code: -38015
+          message: Block gas limit exceeded by the block's transactions
+        - code: -38020
+          message: Block number in sequence did not increase
+        - code: -38021
+          message: Block timestamp in sequence did not increase
+        - code: -38022
+          message: MoveToAddress referenced itself in replacement
+        - code: -38023
+          message: Multiple MoveToAddress referencing the same address to replace
+CallResultLog:
+  title: log
+  type: object
+  required:
+    - logIndex
+    - blockhash
+    - blockNumber
+    - address
+    - data
+    - topics
+  properties:
+    logIndex:
+      title: log index
+      $ref: '#/components/schemas/uint256'
+    blockHash:
+      title: block hash
+      $ref: '#/components/schemas/hash32'
+    blockNumber:
+      title: block number
+      $ref: '#/components/schemas/uint64'
+    address:
+      title: address
+      description: When trace transfers is enabled, this field is address(0x0) for ETH transfers.
+      $ref: '#/components/schemas/address'
+    data:
+      title: data
+      $ref: '#/components/schemas/bytes'
+    topics:
+      title: topics
+      type: array
+      items:
+        $ref: '#/components/schemas/bytes32'
+GenericCallTransaction:
```

Comment:
why not to use [#/components/schemas/GenericTransaction](https://github.com/ethereum/execution-apis/blob/ff43500e653abde45aec0f545564abfb648317af/src/schemas/transaction.yaml#L274) - it has same fields...

Reply:
We could use the same, but its missing default values